### PR TITLE
Screenshot share

### PR DESCRIPTION
**Pull request by OBESTMAN**

### CODE REVIEW

File: lib/screens/match/match_list_screen.dart [hunk 1]
```diff
@@ -77,23 +77,23 @@ class _MatchListScreenState extends State<MatchListScreen> {
...
-                 selectedDay = date;
-                 print(date);
-               });
-               getMatches();
-             },
-             containerDecoration: BoxDecoration(color: Colors.white),
-             monthNameWidget: (monthLabel) =>
-                 Container(child: Text(monthLabel), padding: EdgeInsets.only(top: 7, bottom: 3)),
-           ),
+//           CalendarStrip(
```
Comment (Ingenious-App-Studios) [inline lib/screens/match/match_list_screen.dart:80]: commented

File: lib/screens/profile/search_friends.dart [hunk 2]
```diff
@@ -36,58 +36,59 @@ class _FindFriendsPageState extends State<FindFriendsPage> {
...
-               users = res;
-               users.sort(
-                   (a, b) => a['username'].toString().toLowerCase().compareTo(b['username'].toString().toLowerCase()));
-             });
-           },
-           decoration:
-               InputDecoration.collapsed(hintText: "Click here to search", hintStyle: GoogleFonts.roboto(color: Colors.black)),
-         ),
+          child: Container(),
+//          FloatingSearchBar.builder(
```
Comment (Ingenious-App-Studios) [inline lib/screens/profile/search_friends.dart:40]: commented

File: lib/widgets/text_field_with_date_picker.dart [hunk 3]
```diff
@@ -48,13 +48,13 @@ class TextFieldWithDatePicker extends StatelessWidget {
...
               color: Colors.grey,
               onPressed: () {
-                DatePicker.showDatePicker(context,
-                    showTitleActions: true,
-                    minTime: DateTime(1940, 1, 1), onChanged: (date) {
-                  print('change $date');
-                }, onConfirm: (date) {
-                  onDateChanged(date);
-                });
+//                DatePicker.showDatePicker(context,
```
Comment (Ingenious-App-Studios) [inline lib/widgets/text_field_with_date_picker.dart:51]: commented

File: pubspec.yaml [hunk 4]
```diff
@@ -18,17 +18,17 @@ dependencies:
   modal_progress_hud: ^0.1.3
   image_picker: ^0.6.3+4
   http: ^0.12.0+4
-  flutter_datetime_picker: ^1.3.4
+#  flutter_datetime_picker: ^1.3.4
```
Comment (Ingenious-App-Studios) [inline pubspec.yaml:21]: uncomment all required packages

**Review by Ingenious-App-Studios — CHANGES_REQUESTED** [review]

File: lib/screens/match/match_list_screen.dart (hunk 1 above)
Comment (OBESTMAN) [reply lib/screens/match/match_list_screen.dart]: resolved

File: lib/screens/profile/search_friends.dart (hunk 2 above)
Comment (OBESTMAN) [reply lib/screens/profile/search_friends.dart]: resolved

**Pull request merged**